src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportFAT2FrameworkToPL.sql

```diff
@@ -0,0 +1,144 @@
+﻿
+CREATE PROCEDURE [dbo].[ImportFAT2FrameworkToPL]
+(
+   @RunId int
+)
+AS
+-- ==========================================================================================================
+-- Author:      Sarma Evani
+-- Create Date: 17/Dec/2020
+-- Description: Import, Transform and Load FAT2 Framework Presentation Layer Table
+-- ==========================================================================================================
+BEGIN TRY
+		DECLARE @LogID int
+
+		/* Start Logging Execution */
+
+			  INSERT INTO Mgmt.Log_Execution_Results
+				  (
+					RunId
+				   ,StepNo
+				   ,StoredProcedureName
+				   ,StartDateTime
+				   ,Execution_Status
+				  )
+			  SELECT 
+					@RunId
+				   ,'Step-6'
+				   ,'ImportFAT2FrameworkToPL'
+				   ,getdate()
+				   ,0
+
+			  SELECT @LogID=MAX(LogId) FROM Mgmt.Log_Execution_Results
+			  WHERE StoredProcedureName='ImportFAT2FrameworkToPL'
+			  AND RunId=@RunID
+
+			BEGIN TRANSACTION
+
+					DELETE FROM [ASData_PL].[FAT2_Framework]
+
+					INSERT [ASData_PL].[FAT2_Framework]
+					(												
+							[Id],
+							[FrameworkOccupationId],
```

Comment:
Can we call it VaFrameworkOccupationId instead?

Reply:
Changed the column to VaFrameworkOccupationId